webroot/js/sentences.link.js

```diff
@@ -97,6 +97,30 @@ function linkToSentence(sentenceId, langFilter) {
         linkTo.hide();
     } else {
         linkTo.show();
-        $("#linkToSentence" + sentenceId).focus();
+        var inputField = $("#linkToSentence" + sentenceId);
+
+        navigator.clipboard && navigator.clipboard.readText().then(clipText => {

```

Comment:
Welcome to the dev side of Tatoeba and thanks for taking care of implementing your own feature request :)

I have one issue on my side: on Firefox, this triggers the error `TypeError: navigator.clipboard.readText is not a function` and the rest of the code is not executed so the input is no longer focused upon clicking on the link icon.

Reply:
How unlucky!
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText says:

> Firefox only supports reading the clipboard in browser extensions, using the "clipboardRead" extension permission.

That means that Firefox essentially doesn't support reading from clipboard for Tatoeba, which leaves only Chrome and Opera that will benefit from this enhancement.

Reply:
The commit below fixes the input focus issue with Firefox.